package.json

```diff
@@ -47,10 +47,6 @@
 	"ava": {
 		"files": [
 			"test/lib/**/*.js"
-		],
-		"sources": [
```

Comment:
Double check whether this will now also watch the `test/tmp/*` files generated during test execution